clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/ClioDockerComposeContainer.scala

```diff
@@ -27,7 +28,11 @@ class ClioDockerComposeContainer(
   elasticsearchHostname: String,
   exposedServices: Map[String, Int],
   seededDocuments: Map[ElasticsearchIndex[_], Seq[ClioDocument]] = Map.empty
-) extends DockerComposeContainer(composeFile, exposedServices) {
+) extends DockerComposeContainer(
+      Seq(composeFile),
+      exposedServices,
+      Base58.randomString(6).toLowerCase()
```

Comment:
Testcontainers provides an `apply` method on the `DockerComposeContainer` which does this transformation (wrapping the file in a `Seq`, and generating a random name), but you can't extend an `apply` method...